workerbee/base.py

```diff
@@ -1,6 +1,56 @@
-from functools import partial
+import datetime
+import logging
 import random
+import time
 import sys
+from functools import partial
+
+
+DEFAULT_LOGGER = logging.getLogger('workerbee_default')
```

Comment:
It can be whatever you want and you can choose the pattern with the formatter. If all that information offends you you can change the pattern.